deploy/k8s/setup.sh

```diff
@@ -4,7 +4,7 @@ set -e
 cd "$(dirname "$0")"
 ./check.sh
 
-: ${NUODB_CP_VERSION:="2.4.1"}
+: ${NUODB_CP_VERSION:="2.5.0"}
```

Comment:
This change actually was not needed in order to enable testing of 2.5.0. I only made this change so that if someone invokes the script directly, they use the latest version. This change has no effect on the CircleCI runs whatsoever and is only for convenience.